dev/InfoBar/InfoBar_themeresources.xaml

```diff
@@ -70,54 +86,103 @@
+    <x:Double x:Key="InfoBarIconFontSize">16</x:Double>
 
-    <Thickness x:Key="InfoBarPanelMargin">0,0,12,0</Thickness>
+    <Thickness x:Key="InfoBarPanelMargin">0,0,16,0</Thickness>
     <Thickness x:Key="InfoBarPanelHorizontalOrientationPadding">0,0,0,0</Thickness>
-    <Thickness x:Key="InfoBarPanelVerticalOrientationPadding">0,10,0,10</Thickness>
+    <Thickness x:Key="InfoBarPanelVerticalOrientationPadding">0,14,0,18</Thickness>
 
-    <Thickness x:Key="InfoBarTitleHorizontalOrientationMargin">0,10,0,0</Thickness>
-    <Thickness x:Key="InfoBarTitleVerticalOrientationMargin">0,10,0,0</Thickness>
+    <Thickness x:Key="InfoBarTitleHorizontalOrientationMargin">0,14,0,0</Thickness>
+    <Thickness x:Key="InfoBarTitleVerticalOrientationMargin">0,14,0,0</Thickness>
 
-    <Thickness x:Key="InfoBarMessageHorizontalOrientationMargin">8,10,0,0</Thickness>
+    <Thickness x:Key="InfoBarMessageHorizontalOrientationMargin">12,14,0,0</Thickness>
     <Thickness x:Key="InfoBarMessageVerticalOrientationMargin">0,4,0,0</Thickness>
 
-    <Thickness x:Key="InfoBarActionHorizontalOrientationMargin">12,8,0,0</Thickness>
+    <Thickness x:Key="InfoBarActionHorizontalOrientationMargin">16,8,0,0</Thickness>
     <Thickness x:Key="InfoBarActionVerticalOrientationMargin">0,12,0,0</Thickness>
 
-    <x:Double x:Key="InfoBarActionButtonMinWidth">96</x:Double>
-    <x:Double x:Key="InfoBarActionButtonMinHeight">24</x:Double>
-    <Thickness x:Key="InfoBarActionButtonPadding">8,0,8,1</Thickness>
-    <StaticResource x:Key="InfoBarActionButtonCornerRadius" ResourceKey="ControlCornerRadius"/>
-
-    <x:Double x:Key="InfoBarHyperlinkButtonFontSize">14</x:Double>
-    <x:Double x:Key="InfoBarHyperlinkButtonMinHeight">24</x:Double>
-    <Thickness x:Key="InfoBarHyperlinkButtonPadding">0,0,0,1</Thickness>
-
     <Symbol x:Key="InfoBarCloseButtonSymbol">Cancel</Symbol>
-    
-    <Style x:Key="InfoBarCloseButtonStyle" TargetType="Button">
+
+    <Style x:Key="InfoBarCloseButtonStyle" TargetType="Button" BasedOn="{StaticResource DefaultButtonStyle}">
         <Style.Setters>
             <Setter Property="Width" Value="{StaticResource InfoBarCloseButtonSize}"/>
             <Setter Property="Height" Value="{StaticResource InfoBarCloseButtonSize}"/>
             <Setter Property="VerticalAlignment" Value="Top"/>
-            <Setter Property="Background" Value="Transparent"/>
-            <Setter Property="BorderBrush" Value="Transparent"/>
-            <Setter Property="Margin" Value="4"/>
+            <Setter Property="Background" Value="{ThemeResource AppBarButtonBackground}" />
+            <Setter Property="Foreground" Value="{ThemeResource AppBarButtonForeground}" />
+            <Setter Property="BorderBrush" Value="{ThemeResource AppBarButtonBorderBrush}" />
+            <Setter Property="Margin" Value="5"/>
             <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
+            <Setter Property="Template">
```

Comment:
to only display the close glyph, without the border and fills the new Button control has?

Reply:
To change the foreground/background/border brushes.

Reply:
Doesn't the foreground background and border fill of the button reference theme resources? can't we just override those theme resource instead of providing a whole new template?

Reply:
So in this case in the CloseButton.Resources we could, for example, add 
`<StaticResource x:key="ButtonForegroundPointerOver" value="AppBarButtonForegroundPointerOver"/>` instead of overriding the template entirely

Reply:
Fixed!